pom.xml

```diff
@@ -74,10 +74,9 @@
         <powerauth-crypto.version>1.5.0-SNAPSHOT</powerauth-crypto.version>
 
         <!-- Library Versions -->
-        <guava.version>31.1-jre</guava.version>
```

Comment:
Obsolete declaration, most probably to override some CVE. Transitive dependency of `google-api-client` anyway.